sass/components/_channel-header__icon.scss

```diff
@@ -52,4 +52,8 @@
 .channel-header__icon--wide {
     width: auto;
     padding: 0 6px;
-}
\ No newline at end of file
+}
+
+.userGuideHelp {
+    display: inline-table;
```

Comment:
Hmm, this is really hacky, but I don't see a better way short of replacing this with the new upcoming menu common component. 😛